tests/e2e_regression/e2e_config_2.yml

```diff
@@ -0,0 +1,34 @@
+analysis:
```

Comment:
A readme in `e2e_regression` folder or a doc comment in e2e_config_1.yml and e2e_config_2.yml might be useful explaining why two are needed/used. I assume first _1 runs the block analyzers and once it's done the _2 is used to run the non-block analyzers.

Reply:
Ah that's actually part of Andy's #525. My PR was based on top of it, but GitHub lost track of it when Andy rebased. Sorry this has made for a bulkier and more confusing review.
Your point stands though; I added comments directly in the .yml, thank you.

Reply:
Thanks! :+1: